sequencer/src/api/data_source.rs

```diff
@@ -216,7 +217,6 @@ pub struct PublicHotShotConfig {
     num_nodes_with_stake: NonZeroUsize,
     known_nodes_with_stake: Vec<PeerConfig<PubKey>>,
     known_da_nodes: Vec<PeerConfig<PubKey>>,
-    my_own_validator_config: PublicValidatorConfig,
```

Comment:
Have we removed private stuff from the `HotShotConfig`? If so we can delete this whole struct and just publish the `HotShotConfig` directly from API endpoints. Although maybe it's still good to have this explicit so we don't shoot ourselves in the foot later by adding something private to `HotShotConfig`...

Reply:
Yeah we did remove private stuff. Let me see if I can get rid of this, I don't think we should put anything private in the "network" config

Reply:
Oh, right now we still have some sensitive info in there (`manual_start_password`)? So I will have a separate PR to merge everything into one once we remove that from the "network" config